pkg/apis/triggers/v1alpha1/interceptor_types.go

```diff
@@ -110,6 +110,9 @@ type InterceptorRequest struct {
 	// Header are the headers for the incoming HTTP event
 	Header map[string][]string `json:"header,omitempty"`
 
+	// Form is the form data for the incoming HTTP event
+	Form map[string][]string `json:"form,omitempty"`
```

Comment:
Here Form is of type `map[string][]string` but in v1beta1/interceptor_types.go its `string`
Is it intentional :thinking:

Reply:
no. ill fix it.